src/patternfly/demos/PageLayout/page-template-header-elements.hbs

```diff
@@ -0,0 +1,31 @@
+{{#> page-header-tools}}
+  {{#> toolbar}}
+    {{#> toolbar-group pf-l-toolbar__group--modifier="pf-u-sr-only pf-u-visible-on-lg"}}
+      {{#> toolbar-item}}
+        {{#> button btnClass="pf-m-plain pf-m-lg" btnAttributes='aria-label="Overflow actions"'}}
+          <i class="fas fa-bell"></i>
+        {{/button}}
+      {{/toolbar-item}}
+      {{#> toolbar-item}}
+        {{#> button btnClass="pf-m-plain pf-m-lg" btnClass="pf-m-action" btnAttributes='aria-label="Overflow actions"'}}
+          <i class="fas fa-cog"></i>
+        {{/button}}
+      {{/toolbar-item}}
+    {{/toolbar-group}}
+    {{#> toolbar-group}}
+      {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-hidden-on-lg pf-u-mr-0"}}
+        {{#> button btnClass="pf-m-plain pf-m-lg" btnAttributes='aria-label="Overflow actions"'}}
+          <i class="fas fa-ellipsis-v"></i>
+        {{/button}}
+      {{/toolbar-item}}
+      {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-sr-only pf-u-visible-on-lg"}}
+        {{#> dropdown pf-c-dropdown--Modifier="pf-m-plain" id="dropdown-example-collapsed" dropdown--attribute="pf-m-plain" pf-c-dropdown--HasToggleIcon="true"}}
+          Kyle Baker
+        {{/dropdown}}
+      {{/toolbar-item}}
+    {{/toolbar-group}}
+  {{/toolbar}}  
+  <div class="pf-l-page__header-avatar">
```

Comment:
I don’t know about `pf-l-page__header-avatar` since by the layouts should not create styles for things, just place them, maybe we need a `pf-c-avatar` and the page layout will place it.

Reply:
Updated